src/Microsoft.VisualStudio.Threading/AsyncReaderWriterLock.cs

```diff
@@ -339,6 +371,11 @@ protected bool CaptureDiagnostics
             set { this.captureDiagnostics = value; }
         }
 
+        /// <summary>
+        /// Gets a time delay to check whether pending writer lock and reader locks forms a deadlock.
+        /// </summary>
+        protected virtual TimeSpan DeadlockCheckTimeOut => DefaultDeadlockCheckTimeOut;
```

Comment:
`Timeout` is also a discrete term. It's conventional to _not_ capitalize the `O`.

Reply:
fixed